app/models/version/project_sharing.rb

```diff
@@ -120,7 +120,7 @@ def project_sharing_join_condition(sharing_table, projects_table)
     when 'hierarchy'
       project_sharing_hierarchy_join_condition(sharing_table, projects_table)
     when 'system'
-      Arel::Nodes::True.new
+      "1=1"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.